libraries/AP_HAL/UARTDriver.h

```diff
@@ -61,8 +63,8 @@ class AP_HAL::UARTDriver : public AP_HAL::BetterStream {
     virtual int16_t read_locked(uint32_t key) { return -1; }
     
     // control optional features
-    virtual bool set_options(uint16_t options) { return options==0; }
-    virtual uint8_t get_options(void) const { return 0; }
+    virtual bool set_options(uint16_t options) { _last_options = options; return options; }
```

Comment:
You've inverted the response of this method.

Somebody better have a good reason for this not being `void`....

Reply:
Fixed